force-app/main/default/lwc/formulaBuilder/formulaBuilder.js-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="helloWorld">
+  <apiVersion>46.0</apiVersion>
+  <isExposed>true</isExposed>
+  <targets>
+    <target>lightning__AppPage</target>
+    <target>lightning__RecordPage</target>
+    <target>lightning__HomePage</target>
```

Comment:
go ahead and Flow enable this, per https://unofficialsf.com/adding-lightning-web-components-to-flow-screens/ by adding a target and targetconfigs

Reply:
@alexed1 I added target, but not added targetconfigs, because we use list strings or objects and I think we do not need to use design fields to pass those parameters.